apps/web/package.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "scripts": {
     "dev": "next dev --turbo",
-    "build": "next build",
+    "build": "next build && pnpm sitemap",
```

Comment:
Originally, it didn't appear that the `postbuild` script was being run automatically. I tried converting it to just echo something out, and it never showed up. I also noticed that sitemap files weren't being generated in my preview deploy. 

Is there a better way to ensure that the `sitemap` command is being run?

Reply:
For pnpm we'd have to [enable this functionality](https://pnpm.io/cli/run#enable-pre-post-scripts).

We could add an `.npmrc` with the following contents:

```ini
enable-pre-post-scripts=true
```

I think that having it like you have it in the `build` script explicitly is better, though!